schemas/v3.1/schema.yaml

```
@@ -0,0 +1,809 @@
+$id: 'https://spec.openapis.org/oas/3.1/schema/2021-02-18'
+$schema: 'https://json-schema.org/draft/2020-12/schema'
+type: object
+required:
+  - openapi
+  - info
+anyOf:
+  - required:
+      - components
+  - required:
+      - paths
+  - required:
+      - webhooks
+allOf:
+  - $ref: '#/$defs/Mixins/$defs/Extensible'
+properties:
+  openapi:
+    type: string
+    pattern: ^3\.1\.\d(-.+)?$
+  jsonSchemaDialect:
+    type: string
+    format: uri
+  info:
+    $ref: '#Info'
+  externalDocs:
+    $ref: '#ExternalDocumentation'
+  servers:
+    type: array
+    items:
+      $ref: '#Server'
+  security:
+    type: array
+    items:
+      $ref: '#SecurityRequirement'
+  tags:
+    type: array
+    items:
+      $ref: '#Tag'
```

```suggestion
      $ref: '#Tag'
    uniqueItems: true
```
As per:
> Each tag name in the list MUST be unique.

and to match the [3.0 schema](https://github.com/OAI/OpenAPI-Specification/blob/master/schemas/v3.0/schema.yaml#L29).